proto/rill/admin/v1/api.proto

```diff
@@ -2126,7 +2126,7 @@ message GetBookmarkResponse {
 message CreateBookmarkRequest {
   string display_name = 1;
   string description = 2;
-  bytes data = 3;
+  string url_search = 9;
```

Comment:
I'm not sure what our policy is on breaking API changes. Might we want to retain the `data` field to not break old clients during the rollout?

Reply:
Hmmm not sure how lack of validation in the backend makes it different for `url_search` vs `state`. These are actually url search params, UI will have to handle it differently if they are not. So lets keep it as `url_search`, other variants for non-dashboard bookmarks can have different fields in the future. Note that this matches 1-1 with bookmark resource in ListBookmarks or GetBookmark.

Reply:
Sounds good to me 👍